file-formats/zif_aff_types_v1.intf.abap

```diff
@@ -6,25 +6,37 @@ INTERFACE zif_aff_types_v1 PUBLIC.
   "! The ABAP file format version
   TYPES ty_format_version TYPE string.
 
-  "! <p class="shorttext">ABAP Language Version</p>
+  "! <p class="shorttext">ABAP Language Version (Non-Source Code Objects)</p>
```

Comment:
We shouldn't add this to the title. I thought we remove it from the titles. If would be inherited by object types which reuse `ty_abap_language_version` 
```suggestion
  "! <p class="shorttext">ABAP Language Version</p>
```

If wanted, we can keep it in the description.